spec/fixtures/files/mappers/release_6_1/anthro/anthro_4_1_0-osteology.json

```diff
@@ -8,7 +8,7 @@
     "profile_basename": "anthro",
     "ns_uri": {
       "osteology_common": "http://collectionspace.org/services/osteology",
-      "osteology_anthropology": null
```

Comment:
This was causing the major initial error. I manually fixed the RecordMapper for osteology in the mapper fixtures. I made an issue [here](https://github.com/collectionspace/cspace-config-untangler/issues/30) to fix the creation of the RecordMapper. I made Issue #63 to handle this failure more gracefully.